config/bidderinfo.go

```diff
@@ -351,7 +379,11 @@ func validateSyncer(bidderInfo BidderInfo) error {
 
 func applyBidderInfoConfigOverrides(configBidderInfos BidderInfos, fsBidderInfos BidderInfos) (BidderInfos, error) {
 	for bidderName, bidderInfo := range configBidderInfos {
-		if fsBidderCfg, exists := fsBidderInfos[bidderName]; exists {
+		normalizedBidderName, bidderNameExists := openrtb_ext.NormalizeBidderName(bidderName)
+		if !bidderNameExists {
+			return nil, fmt.Errorf("error finding bidder in bidders list by bidder name loaded in configs for bidder %s. Configs include data in environment variables and pbs.json or pbs.yaml files", bidderName)
```

Comment:
Nitpick: Suggested wording simplification (and similarly for the other error message):
`error setting configuration for bidder %s: unknown bidder`